content/docs/command-reference/cache/index.md

```diff
@@ -1,37 +0,0 @@
-# cache
-
-Contains a helper command to set the <abbr>cache</abbr> directory location:
```

Comment:
BTW another kind of link to review is those done automatically with the `` `dvc cache` `` text in other md files. Those used to link here for an explanation of what the cache is, but should be regular links to the concept now, instead (same for `dvc params`). Thanks

Reply:
\* same for SOME INSTANCES of `dvc params` — need to check the context.

Reply:
Nice... OK, fixed one `dvc cache` link in the docs. Should I fix the one in this blog post too?:

> /blog/2020-04-16-april-20-community-gems.md

Reply:
> * same for SOME INSTANCES of dvc params — need to check the context.

OK, got these too. Didn't change the `dvc params diff` instances.

Reply:
> Should I fix the one in this blog post too?:
/blog/2020-04-16-april-20-community-gems.md

I don't see any `dvc cache` link in https://dvc.org/blog/april-20-community-gems

Reply:
Links will be tricky. This is also why we need to split this PR into many so we can take a look at the details concept by concept. Resolvin this one for nwo.